gent_base.libraries.yml

```diff
@@ -66,7 +66,7 @@ facet_range_slider:
 checkbox_filter:
   version: 1.x
   js:
-    build/styleguide/js/checkbox_filter.functions-min.js: { minified: true }
+    build/styleguide/js/checkbox_dynamic.functions-min.js: { minified: true }
     source/js/checkbox_filter.bindings.js: {}
```

Comment:
Kan hiervoor ook niet de minified versie uit build dir gebruikt worden?

Reply:
Geen enkele binding file wordt uit build gehaald zo te zien, de reden hiervoor is me ook nog niet duidelijk. Maar te gevaarlijk om dit nu zomaar aan te passen